Emby.Server.Implementations/Updates/InstallationManager.cs

```diff
@@ -129,28 +118,24 @@ private void OnPluginInstalled(PackageVersionInfo package)
             IJsonSerializer jsonSerializer,
             IServerConfigurationManager config,
             IFileSystem fileSystem,
-            ICryptoProvider cryptographyProvider,
```

Comment:
So the crypto provider provides hashing functionality, while it isn't currently implemented, it might be wise to keep it around so we could do hash verification of plugins.

Reply:
You can just add it back when you need it.